pkg/kubelet/network/hairpin/hairpin.go

```diff
@@ -37,7 +37,7 @@ const (
 )
 
 var (
-	ethtoolOutputRegex = regexp.MustCompile("peer_ifindex: (\\d+)")
+	ipLinkOutputRegex = regexp.MustCompile("eth0@if(\\d+)")
```

Comment:
Right now eth0 is hardcoded in an other file with a todo to make it configurable. I may fetch the value from there instead.

Do you still think it may be something other than eth0 (on kubenet for example, which I haven't tested much)